hack/cronjob-update-mapping.sh

```diff
@@ -16,7 +16,7 @@ git checkout -b update
 
 # Generate and push mapping
 ## OCP Engineering Mappings
-ci-test-mapping map --mode=bigquery --push-to-bigquery
+ci-test-mapping map --mode=bigquery --push-to-bigquery --map-variant
```

Comment:
Could we also produce this data for QE? Even if it's empty I want to make sure the tables, etc get created (line #21)